src/iptax/cli/elements.py

```
@@ -0,0 +1,177 @@
+    # Display summary
+    repositories = {change.repository.get_display_name() for change in changes}
+    console.print("\n[bold]Summary:[/bold]")
+    console.print(f"  Total changes: {len(changes)}")
+    console.print(f"  Repositories: {len(repositories)}")
+
+
+def display_review_results(
+    console: Console,
+    judgments: list[Judgment],
+    changes: list[Change],
+    accepted: bool,
```

Ruff (0.14.7)

Boolean-typed positional argument in function definition

(FBT001)